README.md

```diff
@@ -1,11 +1,27 @@
 # bpmn-visualization 'getting started' tutorial
 
-A getting started demo that uses the [bpmn-visualization](https://github.com/process-analytics/bpmn-visualization-js) library to visualize runtime monitoring data ⏲️.
+This is a template bootstrap. You can clone it and use it to follow the [getting started tutorial](https://medium.com/@process-analytics/getting-started-with-bpmn-visualization-a54fbafae2df).
 
-The code of this tutorial is released under the [Apache 2.0](LICENSE) license.

```

Comment:
we must keep this information

Reply:
fixed in 20ed06b8c394193819decc5d6e2c51de18add023